src/handlers/DibiHandler.php

```diff
@@ -110,12 +111,14 @@ public function getInfo()
 			return NULL;
 		}
 
-		$query = 'EXPLAIN (FORMAT JSON) ' . $this->event->sql;
-		$explain = $this->event->connection->nativeQuery($query);
-		$data = json_decode($explain->fetchSingle(), TRUE);
-		return Html::el()->setHtml(Dumper::toHtml($data, [
-			Dumper::COLLAPSE => TRUE,
-		]));
+		if(\Nette\Utils\Strings::startsWith($this->event->sql, 'SELECT')) {
```

Comment:
This is definitely not correct, select is not the only keyword _explain-able_ queries may start with (recall the `with … select …` syntax, for example, or `explain analyze execute`). Besides, there may probably be whitespace before the query.